phone-test/src/pages/CallsList/CallsList.tsx

```diff
@@ -36,15 +38,20 @@ const CallsListPage = () => {
   const [totalFilteredCalls, setTotalFilteredCalls] = useState(0);
   const [paginatedCalls, setPaginatedCalls] = useState<CallGroup[]>([]);
 
-  const { loading, error, data } = useQuery(PAGINATED_CALLS, {
+  const { loading, error, data, subscribeToMore } = useQuery(PAGINATED_CALLS, {
     variables: { offset: 0, limit: 200 }
   });
```

Comment:
Made this change from the original version:
```
 const { loading, error, data, subscribeToMore } = useQuery(PAGINATED_CALLS, {
     variables: {
       offset: (activePage - 1) * pageSize,
       limit: pageSize
     }
});
```
This is a hack, due to not being able to apply a sort on the `PAGINATED_CALLS` query. Basic observation of the API response led me to think that the results are not consistent, i.e, the order of the calls are returned arbitrarily and for the implementation I had in mind, of sorting and grouping the calls by date, the solution I found, not being able to change the gql code on the backend, was to retrieve 200 results(this is an arbitrary value, which seemed to return the maximum amount of calls available for display) and perform the sorting and grouping operations on the frontend. 
In my opinion, this is not ideal at all, as first of all, it may miss calls, due to the hardcoded fetch value of 200, and if we are in a scenario, in which a user might have thousands of call, I believe we may run into performance issues on the frontend.
Part of the thoughts in this comment, should be applied on the `challenge` PR, on which this work is based, but I'm including it here